cmake/environment/GithubSshTest.cmake

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+#
+# medInria
```

Comment:
the purpose of a PR is to avoid pushing a WIP code in the master to the master stays clean.

once merged we have no diff and from my experience crap remains...

please let's merge a clean PR so we can actually advertise our master branch for a release